include/aspect/simulator/assemblers/interface.h

```diff
@@ -160,6 +167,15 @@ namespace aspect
 
           FEFaceValues<dim> face_finite_element_values;
 
+          using StokesPreconditioner<dim>::reinit;
+
+          void reinit (const typename DoFHandler<dim>::active_cell_iterator &cell_ref, const unsigned face_number_ref)
```

Comment:
same here for the body of the function.

Break the two arguments of the function onto separate lines.